test/server/repositories/UserRepository.test.ts

```diff
@@ -235,59 +242,27 @@ describe('UserRepository', () => {
       await expect(userRepo.findUrlsForUser(conditions)).rejects.toBeInstanceOf(
         NotFoundError,
       )
-      expect(scope).toHaveBeenCalledWith([
-        'defaultScope',
-        {
-          method: ['urlsWithQueryConditions', conditions],
-        },
-      ])
+      expect(scope).toHaveBeenCalledWith(['defaultScope', 'getClicks'])
     })
 
     it('throws NotFoundError on findAndCountAll without user', async () => {
       findAndCountAll.mockResolvedValue({ rows: [], count: 0 })
       await expect(userRepo.findUrlsForUser(conditions)).rejects.toBeInstanceOf(
         NotFoundError,
       )
-      expect(scope).toHaveBeenCalledWith([
-        'defaultScope',
-        {
-          method: ['urlsWithQueryConditions', conditions],
-        },
-      ])
-    })
-
-    it('returns empty result on user without urls', async () => {
-      const rows = [{ Urls: [] }]
-      findAndCountAll.mockResolvedValue({ rows, count: rows.length })
-      await expect(userRepo.findUrlsForUser(conditions)).resolves.toStrictEqual(
-        {
-          urls: [],
-          count: 0,
-        },
-      )
-      expect(scope).toHaveBeenCalledWith([
-        'defaultScope',
-        {
-          method: ['urlsWithQueryConditions', conditions],
-        },
-      ])
```

Comment:
was this test removed?
also thinking if we can test for some functionalities in `findUrlsForUser`, like the tags being fetched and returned along with the URLs!

Reply:
done